src/Symfony/Component/Form/Command/FormDebugCommand.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Form\FormRegistryInterface;
+use Symfony\Component\Form\ResolvedFormTypeInterface;
+use Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A console command for retrieving information about form types.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class FormDebugCommand extends Command
+{
+    private $formRegistry;
+    private $optionsResolver;
+    private $parents = array();
```

Comment:
I'm thinking in another way to collect the options and classes in an easier way... what do you think if we add an `OptionsResolverWrapper` to do that? also, it would allow us collect more info about options like default values, allowed types, etc.